vars/utils.groovy

```diff
@@ -47,7 +47,7 @@ def buildSchema(){
 def getDockerName(backend)
 {
     def docker_registry = "${headnode}:5000"
-    def tuna_docker_name = "${docker_registry}/ci-tuna:${branch_id}_${backend}_${env.BUILD_ID}"
```

Comment:
we dont want to use the env build id to differentiate the dockers?

Reply:
BUILD_ID is baked into branch_id so I removed it here.